scripts/publish-to-unity-package.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 echo Building project...
-pushd src/ChainSafe.Gaming.Unity
+pushd ../src/ChainSafe.Gaming.Unity
```

Comment:
`../` is not needed here tbh.
you still can run script from the root of the repo like:
```shell
$: ./scripts/publish-to-unity-package.sh
```

Reply:
I see it causes errors when we use `./scripts/publish-to-unity-package.sh` from repo root, I've added a directory check and move into scripts folder to avoid this, so this can run successfully in both cases when opened from repo root and/or when opened from scripts